### ISSUE TITLE
Role Diversity

### ISSUE THREAD

**Issue by ecbrown1** [drnikki/open-demographics#69]
Just wanted to add role diversity as a possible metric to consider.

**Comment by patcon** [comment]
I like the sounds of this! But also, realized I'm not totally sure what "role diversity" means to others. Can you share how you intended it?

**Comment by ecbrown1** [comment]
Absolutely. I intended to shed light on the actual role of the attendees in an organizational structure or hierarchy and the importance of their input. I hope this helps.

**Comment by patcon** [comment]
ah ok, thanks for clarifying, Emily!

But also, with apologies if I seem dense... still unsure whether "diversity" is used in the equity sense, in reference to under-represented groups in roles? Or alternatively used in a purely org management or org structure sense, where having more roles of varying sorts is more "role diversity" than a smaller number of roles (e.g., https://www.jstor.org/stable/2391180?seq=1).

Heh sorry, just wouldn't want this issue to stagnate or be in limbo due to any ambiguity :)
